pennylane/devices/tests/test_gates.py

```diff
@@ -18,6 +18,7 @@
 # pylint: disable=no-self-use
 # pylint: disable=too-many-arguments
 # pylint: disable=pointless-statement
+# pylint: disable=unnecessary-lambda-assignment
```

Comment:
Why not solving the issue instead? You just need to change:
```python
my_func = lambda x: x
```
by
```
def my_func(x):
  return x
```
wherever needed.

Reply:
Just realised this is a test file, so it's not that important. But we should push towards using less `# pylint: disable` statements.

Reply:
The pylint `disable` statement is not related to anything I changed, I only updated the line `all_available_ops.add("ControlledQubitUnitary")`. It's related to a section further down that was added over a year ago; I have `pre-commit` installed and pylint insisted I add this statement before committing my change, even though it relates to an unmodified section of the code. 

Since it's a test file and not related to any changes in this PR, I'm going to leave as is.